app/minecraft/mcservcreate.py

```diff
@@ -110,25 +120,40 @@ def mods(): # Downloads mods for the server and places them within the mods fold
         else: #? Runs if custom URL or anything else is entered  
             #! IDK if this works lol
             print("Trying to Download: {}".format(mod_required))
-            c = urllib3.poolmanager()
-            filename = path+path_type+mod_required.split("/")[-1]   
-            with c.request('GET', mod_required, preload_content=False) as res, open(filename, 'wb') as out_file:
-                shutil.copyfileobj(res, out_file)
-                out_file.close()  
-            time.sleep(2)
-        
-    eulagen()
-                
+            try:
+                c = urllib3.poolmanager()
+                filename = path+path_type+mod_required.split("/")[-1]   
+                with c.request('GET', mod_required, preload_content=False) as res, open(filename, 'wb') as out_file:
+                    shutil.copyfileobj(res, out_file)
+                    out_file.close()  
+                time.sleep(2)
+                print("downloaded:{}".format(mod_required))
+            except:
+                print("Something went wrong, probably my code")
+
+
+def initilise_server_files() # just runs the jar file first to initialise all requried files before server starts.
+    subprocess.call(['java', '-jar', 'server{}fabric.jar'.format(path_type)])
+
+
 def eulagen(): #Auto-completes the minecraft eula
```

Comment:
I would be more descriptive with the function handle here,

```suggestion
def accept_eulagen():
```